proto/palomachain/paloma/gravity/msgs.proto

```diff
@@ -169,3 +172,23 @@ message EventOutgoingTxId {
     string message = 1;
     string tx_id = 2;
 }
+
+message MsgUpdateParams {
+  option (cosmos.msg.v1.signer) = "authority";
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // params defines the x/gravity parameters to update.
+  //
+  // NOTE: All parameters must be supplied.
+  Params params = 2 [(gogoproto.nullable) = false];
+  
+  palomachain.paloma.valset.MsgMetadata metadata       = 6 [(gogoproto.nullable) = false];
```

Comment:
Should be `3` instead of `6`.

Reply:
Yeah,I forget to update it .I'll update it now